contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -118,6 +128,9 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuard {
     /// @dev Whether a nullifier hash has been used already. Used to guarantee an action is only performed once by a single person
     mapping(uint256 => bool) public nullifierHashes;
 
+    /// @notice The max gas limit for a PBH multicall transaction
+    uint256 public multicallGasLimit;
```

Comment:
Any thoughts on updating this to `pbhGasLimit`? We can have the builder read this value and use this to enforce gas limit per PBH UserOp as well.